consumer.go

```diff
@@ -464,7 +491,7 @@ func (child *partitionConsumer) handleResponse(response *FetchResponse) error {
 type brokerConsumer struct {
 	consumer         *consumer
 	broker           *Broker
-	input            chan *partitionConsumer
+	input, acks      chan *partitionConsumer
```

Comment:
And to indicate whether the brokerConsumer should continue processing that subscription or if it's been redispatched